tests/introspection/Mac/MacApiCtorInitTest.cs

```diff
@@ -68,6 +68,10 @@ protected override bool Skip (Type type)
 			// [NSWindow initWithContentRect:styleMask:backing:defer]
 			case "MonoMac.AppKit.NSWindow":
 			case "AppKit.NSWindow":
+			// *** -[NSKeyedArchiver dealloc]: warning: NSKeyedArchiver deallocated without having had -finishEncoding called on it.
+			case "MonoMac.Foundation.NSKeyedArchiver":
+			case "Foundation.NSKeyedArchiver":
```

Comment:
Is this just a warning ? i.e. will it cause failures to be reported (crash / exceptions) ?

because this will disable all tests on the type, so fixing a *warning* might just hide future *errors* - seems a bad tradeoff